test-data/unit/check-literal.test

```diff
@@ -694,7 +694,7 @@ y: Foo[Foo]     # E: Literal[...] must have at least one parameter
 [out]
 
 [case testLiteralBadRawExpressionWithBadType]
-# flags: --new-semantic-analyzer
+
```

Comment:
Are these extra empty lines hard to avoid?

Reply:
They are easy to remove but they cause a lot of test failures due to line numbers changing and
1. Fixing them would have caused a lot more churn in the diff
2. I didn't want to fix them :P